apps/heatmap/heatmap.html

```diff
@@ -155,6 +164,12 @@
     <script src="https://unpkg.com/@popperjs/core@2"></script>
     <script src="https://unpkg.com/tippy.js@6"></script>
 
+		<!-- Smartpen -->
+	  <script type="text/javascript" src="../../common/enhance.js"></script>
```

Comment:
@Vedant1202 Not sure where these changes are coming from? This is from the smart pens project?

Reply:
I'm guessing that he's pulled dev changes into his topic branch, but we haven't done that for our topic branch. Is that right, @Vedant1202?

Reply:
Yeah, I guess so.. I'll stop pulling changes from dev since it's a new branch